v3/src/components/graph/graph-registration.ts

```diff
@@ -36,6 +36,10 @@ registerTileContentInfo({
       }]
     }
     return graphTileSnapshot
+  },
+  getTitle: (tile: ITileLikeModel) => {
+    const data = isGraphContentModel(tile?.content) ? tile.content.dataset : undefined
+    return tile.title || data?.name || ""
```

Comment:
I think you want to use the `DataSet`'s `title` (which will default to the `name` if the `_title` hasn't been specified).
```suggestion
    return tile.title || data?.title || ""
```